requirements.txt

```diff
@@ -1,14 +1,21 @@
+Flask==0.11
+Flask-RESTful==0.3.6
+Jinja2==2.9.6
+MarkupSafe==1.0
+SQLAlchemy==1.1.10
+Werkzeug==0.12.2
+aniso8601==1.2.1
 appdirs==1.4.3
+argparse==1.2.1
 click==6.7
-Flask==0.11
 itsdangerous==0.24
-Jinja2==2.9.6
 lifter==0.4.1
-MarkupSafe==1.0
 packaging==16.8
 persisting-theory==0.2.1
+psycopg2==2.7.1
 pyparsing==2.2.0
+python-dateutil==2.6.0
+pytz==2017.2
 six==1.10.0
-SQLAlchemy==1.1.10
 thespian==3.5.2
-Werkzeug==0.12.2
+wsgiref==0.1.2
```

Comment:
BUG: there is not Python 3 version for `wsgiref` because it has been merged in the STDLIB as `http.server`, use `http.server` instead.